source/features/netiquette.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'dom-chef';
+import * as pageDetect from 'github-url-detection';
+import toMilliseconds from '@sindresorhus/to-milliseconds';
+import select from 'select-dom';
+import twas from 'twas';
+import {InfoIcon} from '@primer/octicons-react';
+
+import createBanner from '../github-helpers/banner';
+import features from '../feature-manager';
+import observe from '../helpers/selector-observer';
+import {buildRepoURL} from '../github-helpers';
+import {closedOrMergedMarkerSelector, getLastCloseEvent} from './jump-to-conversation-close-event';
+import selectHas from '../helpers/select-has';
+
+const isClosedOrMerged = (): boolean => select.exists(closedOrMergedMarkerSelector);
+
+/** Returns milliseconds passed since `date` */
+function timeAgo(date: Date): number {
+	return Date.now() - date.getTime();
+}
+
+function getCloseDate(): Date {
+	const datetime = select('relative-time', getLastCloseEvent())!.getAttribute('datetime')!;
+	console.assert(datetime, 'Datetime attribute missing from relative-time');
+	return new Date(datetime);
+}
+
+const threeMonths = toMilliseconds({days: 90});
+
+function addConversationBanner(newCommentActions: HTMLElement): void {
+	if (!isClosedOrMerged()) {
+		return;
+	}
+
+	const closingDate = getCloseDate();
+	if (timeAgo(closingDate) < threeMonths) {
+		return;
+	}
+
+	const ago = <strong>{twas(closingDate.getTime())}</strong>;
+	const newIssue = <a href={buildRepoURL('issues/new/choose')}>new issue</a>;
+
+	newCommentActions.after(
+		createBanner({
+			classes: 'p-2 mb-n2 ml-n2 mr-n2 mt-3 text-small color-fg-muted border-0'.split(' '),
```

Comment:
Other styles considered:

## Too big

<img width="745" alt="Screenshot 1" src="https://user-images.githubusercontent.com/1402241/226546023-9d9cc075-07ce-4cd8-8ffb-b01f382770a3.png">

## Too quiet

<img width="746" alt="Screenshot 3" src="https://user-images.githubusercontent.com/1402241/226546141-eb475526-1520-47f5-822b-49990aa0c8d9.png">

## Good but could be closer

<img width="676" alt="Screenshot 6" src="https://user-images.githubusercontent.com/1402241/226546217-5135ebee-b4b4-45d0-b6bc-6de99e56ba2d.png">


I also considered making it yellow or red depending on how old it is, but I don't want people to disable the feature because it's _too much._

Reply:
Better. Done